src/Workleap.DomainEventPropagation.Subscription/EventPropagationThrottlingOptions.cs

```diff
@@ -0,0 +1,10 @@
+namespace Workleap.DomainEventPropagation;
+
+public class EventPropagationThrottlingOptions
+{
+    internal const string DefaultSectionName = "EventPropagation:ThrottlingOptions";
+
+    public int MaxEventsPerSecond { get; set; }
```

Comment:
Will you provide guidance about defining these numbers in our internal documentation?